MapboxNavigation/NavigationMapView.swift

```diff
@@ -21,6 +23,10 @@ open class NavigationMapView: MGLMapView {
         22: MGLStyleValue(rawValue: 18)
     ]
     
+    let TrafficSevere = UIColor(red:0.54, green:0.06, blue:0.22, alpha:1.0)
+    let TrafficHeavy = UIColor(red:0.91, green:0.20, blue:0.25, alpha:1.0)
+    let TrafficModerate = UIColor(red:0.95, green:0.65, blue:0.31, alpha:1.0)
```

Comment:
Attempted to make the colors public in https://github.com/mapbox/mapbox-navigation-ios/pull/359/commits/682e04181283cc0ec52339b76aa506eae2ffa26f but it's not quite working right. /cc @frederoni for guidance on this.